This patch modifies m3browser to use the new "m3configvars" package.

Index: m3/m3browser/src/Main.m3
===================================================================
--- Main.m3.orig	Mon Feb  5 10:49:57 1996
+++ Main.m3	Tue Sep 24 15:39:45 1996
@@ -13,13 +13,12 @@
 
 IMPORT Text, Rd, Wr, TextRd, Thread, Time, Fmt, IntRefTbl, IntList;
 IMPORT Process, Params, Lex, OS, FloatMode, Word, IntIntTbl, TextIntTbl;
-IMPORT FileWr, OSError, Atom, FmtTime, M3Config, RTParams, RTCollector;
+IMPORT FileWr, OSError, Atom, FmtTime, M3ConfigVars, RTParams, RTCollector;
 IMPORT FS, RefList, IntSeq, XFormat, RefSeq, TextRefTbl, CharMap, IP;
 IMPORT Buf, ID, Wx, MarkUp, CMarkUp, TCPServer, ErrLog, RTutils, RTHeapStats;
 
 CONST
   Title_page = "m3browser.html";
-  SLASH      = M3Config.PATH_SEP;
   StartPage  = "<HTML>\n<HEAD>\n";
   StartTitle = "<TITLE>";
   Body       = "<BODY BGCOLOR=\"#ffffff\" VLINK=\"#006633\">\n";
@@ -28,10 +27,11 @@
   EndPage    = "</BODY>\n</HTML>\n";
 
 VAR (* configuration *)
-  package_root     := M3Config.PKG_USE;
+  SLASH            := M3ConfigVars.PATH_SEP;
+  package_root     := M3ConfigVars.PKG_USE;
   server_machine   :  TEXT; (* initialized in "ParseOptions" *)
   server_socket    := 3829;
-  derived_dirs     := IntList.List1 (ID.Add (M3Config.BUILD_DIR));
+  derived_dirs     := IntList.List1 (ID.Add (M3ConfigVars.BUILD_DIR));
   n_workers        := 3;
   refresh_interval := 30.0d0; (* minutes *)
   start_time       := Time.Now ();
Index: m3/m3browser/src/m3makefile
===================================================================
RCS file: /home/jdp/m3-cvs/m3/m3browser/src/m3makefile,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 m3makefile
--- m3/m3browser/src/m3makefile	1996/09/24 05:21:56	1.1.1.1
+++ m3/m3browser/src/m3makefile	1996/09/24 05:32:39
@@ -6,6 +6,7 @@
 
 m3_option ("-times")
 
+import ("m3configvars")
 import ("libm3")
 import ("tcp")
 import ("m3tools")
